lib/Core/ErrorState.cpp

```diff
@@ -329,6 +322,25 @@ ref<Expr> ErrorState::propagateError(Executor *executor,
     valueErrorMap[instr] = result;
     return result;
   }
+  case llvm::Instruction::GetElementPtr: {
+    // Check if base is symbolic and whether a symbolic error value already
+    // exists for the newly pointed element
+    ref<Expr> newError;
+    if (isInStoredError(result)) {
+      newError = retrieveStoredError(result);
```

Comment:
@Himeshi This is an excellent observation. This part of the code should also rely on the usage of Cell, according to #35 which has not been merged. But that modification will happen at the point of rebasing this branch with the master, in case #35 is merged first, or at the point of rebasing #35 for merging, in case this branch was merged first.

Perhaps this time let's merge this branch first before #35.